openassessment/templates/openassessmentblock/message/oa_message_complete.html

```diff
@@ -1,13 +1,13 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">{% trans "You Have Completed This Assignment" %} </h3>
+    <h5 class="message__title">{% trans "Status" %} </h5>
```

Comment:
The heading for the entire ORA problem is an H3, so this should be an H4.  The steps themselves are also H4 but I'm OK with this being at the same level as the steps.  The ORA problem status, and the steps for the ORA problem, can all be at the same level, H4.  If the individual steps also have statuses, those would be H5